tests/postgres_tests/test_search.py

```diff
@@ -161,6 +161,16 @@ def test_search_with_non_text(self):
         ).filter(search=str(self.crowd.id))
         self.assertSequenceEqual(searched, [self.crowd])
 
+    def test_phrase_search(self):
+        searched = Line.objects.annotate(
+            search=SearchVector('dialogue'),
+        ).filter(search=SearchQuery('burned body his away', phrase=True))
+        self.assertSequenceEqual(searched, [])
+        searched = Line.objects.annotate(
+            search=SearchVector('dialogue'),
+        ).filter(search=SearchQuery('his body burned away', phrase=True))
+        self.assertSequenceEqual(searched, [self.verse1])
```

Comment:
Use `self.subTest()`?

Reply:
I agree with @claudep, I feel like `subTest` is only useful when you are iterating over test parameters. In the case of inlined assertions it's clear which parameters are tested against from a traceback.

Reply:
Fair enough. I guess the common `Line.objects.annotate(…)` could be factored out though?